docs/deployment/plugins/k8s/index.rst

```diff
@@ -418,24 +150,46 @@ Specify plugin configuration
 
     .. tabs::
 
+      .. group-tab:: Demo (sandbox) cluster
```

Comment:
can we have this under "flyte binary" because demo uses flyte binary helm chart? it's also important to specify that one needs to spin up a cluster using `flytectl demo start` after adding this config.

Reply:
I'm not sure demo uses the flyte-binary chart but the flyte-sandbox one? I added instructions to set up the cluster after adding the configs

Reply:
okay, let's just keep it a separate section. have we modified the other sections as well, to keep it in sync?

Reply:
Yes, I did